### ISSUE TITLE
alpine 2.8 support

### ISSUE THREAD

**Issue by aiomaster** [docker-library/ruby#228]
It would be nice to have alpine 3.8 as image base for newer rubies > 2.5
Is there any reason to stick to 3.7?

**Comment by aiomaster** [comment]
Ah ok, I tried to build it on my own and came across that bug:
https://github.com/ruby/openssl/issues/192
In the next patch releases of ruby it could be fixed, so I may try again then.

**Comment by tianon** [comment]
I've found that it works successfully if we switch back from `libressl` to `openssl`, but that causes a huge number of dependency issues with other Alpine packages (since the two can't be co-installed and many other Alpine packages depend on `libressl`), so that's not really a viable solution. :disappointed:

**Comment by aiomaster** [comment]
Yeah let's just wait until this is released: https://bugs.ruby-lang.org/issues/14754

**Comment by aRobinson-R7** [comment]
Hi
This was released a few days ago:
https://justi.cz/security/2018/09/13/alpine-apk-rce.html

With a patch begin sent out before the article was released:
https://alpinelinux.org/posts/Alpine-3.8.1-released.html

Would it be possible for you to use 3.8.1+, if you are already planning to update the base images, to get over this bug?
Thanks for your work!

**Comment by yosifkit** [comment]
@aRobinson-R7 And the security vulnerability was already fixed in the Alpine 3.2 to 3.8 images: https://github.com/docker-library/official-images/pull/4834.

**Comment by ashleyhull-versent** [comment]
I'd assume 2.4.5/2.5.2 is going to be anywhere from 1-3 months?

**Comment by deivid-rodriguez** [comment]
New rubies were just released so now this can be moved forward! :)

**Comment by ashleyhull-versent** [comment]
#241
